system-addon/common/Reducers.jsm

```diff
@@ -135,10 +135,6 @@ function TopSites(prevState = INITIAL_STATE.TopSites, action) {
         return site;
       });
       return Object.assign({}, prevState, {rows: newRows});
-    case at.PLACES_LINK_DELETED:
-    case at.PLACES_LINK_BLOCKED:
-      newRows = prevState.rows.filter(val => val && val.url !== action.data.url);
-      return Object.assign({}, prevState, {rows: newRows});
```

Comment:
So @k88hudson has mentioned how it's nice to have the UI update right away without waiting for Places roundtrip especially on slow machines. Although while waiting for the refresh, there will be fewer than 12 items. until the `refresh` completes... @k88hudson ?

Reply:
Yeah, this has the potential to be pretty slow on some machines, as well as causing some weird effects if the query returns new stuff

Reply:
So, these aren't the user events triggering the delete/block in the first place. These happen when PlacesFeed notices newtabutils has changed something. So keeping this reducer maybe saves the UI time between this action and `TOP_SITES_UPDATED`

Reply:
The user events seem to be `BLOCK_URL and `DELETE_HISTORY_URL` (`DELETE_HISTORY_URL_CONFIRM` and `DELETE_CONFIRM` were added in #2749 but are unused)

Reply:
Ok ,since the broadcasting of updates to other pages is handled by a refresh now, let's just change these to use `BLOCK_URL` and `DELETE_HISTORY_URL` for the local instance only to help with the UI lag.